### ISSUE TITLE
Web Socket connection failure

### ISSUE THREAD

**Issue by vincent314** [rjaros/kvision#164]
Hi,

I am starting using KVision framework with a WebSocket Service. However, I get this error :
`'WebSocket': The URL 'ws://localhost:3000kvws/routeGameServiceManager0' is invalid.` (note the missing `/` after port number).

My guess is that WebSocket URL built by `pl.treksoft.kvision.remote.Utils#getWebSocketUrl` is wrong.

As a workaround, I forced a `/` with an empty url prefix in **index.html** `<head>` !
```
    <script type="text/javascript">
        window["kv_remote_url_prefix"] = "";
    </script>
```

I hope this will help to fix this issue.
Thank you for this exciting Kotlin framework !

Version: 3.10.2
OS: MacOS
Browser: Chrome / Firefox

**Comment by rjaros** [comment]
Thx for feedback!
I confirm that the error is in `getWebSocketUrl` function.

**Comment by rjaros** [comment]
Released with 3.11.0
